File: snippets/run_actor.py
# Run the Airbnb Data Scraper: https://apify.com/factden/airbnb-data-scraper
from apify_client import ApifyClient

client = ApifyClient("YOUR_APIFY_TOKEN")
run = client.actor("factden/airbnb-data-scraper").call(run_input={
    "mode": "market",
    "location": "Lisbon, Portugal",
    "sampleSize": 300,
    "currency": "USD",
})
for item in client.dataset(run["defaultDatasetId"]).iterate_items():
    print(item)

File: snippets/run_actor.js
// Run the Airbnb Data Scraper: https://apify.com/factden/airbnb-data-scraper
import { ApifyClient } from 'apify-client';

const client = new ApifyClient({ token: 'YOUR_APIFY_TOKEN' });
const run = await client.actor('factden/airbnb-data-scraper').call({
    mode: 'market',
    location: 'Lisbon, Portugal',
    sampleSize: 100,
    currency: 'USD',
});
const { items } = await client.dataset(run.defaultDatasetId).listItems();
console.log(items);
